### PR TITLE
258 follow reducer tests closes #258

### LINKED ISSUE #258: Make reducer tests for follow/unfollow action

### PR DESCRIPTION
**Pull request by georgeF105**

### CODE REVIEW

**Comment by kristy-mj** [comment]
Under review

File: client/test/games.js [hunk 1]
```diff
@@ -99,3 +99,49 @@ test('FILTER_FOLLOW_GAMES', (t) => {
...
+  })
+  store.dispatch({
+    gameId,
+    type: gamesActions.FOLLOW_GAME
+  })
+  t.deepEqual(store.getState().games.get('games').toJS(), expected, 'FOLLOW_GAME updated the selected game to followed')
+  t.end()
+})
+
+test('FOLLOW_GAME', (t) => {
```
Comment (kristy-mj) [inline client/test/games.js]: test UNFOLLOW_GAME???

**Comment by georgeF105** [comment]
Fixed thanks Kristy!

**Comment by kristy-mj** [comment]
Good stuff!!

**Pull request merged**